src/gui/editorwidgets/qgshtmlwidgetwrapper.h

```diff
@@ -53,6 +53,12 @@ class GUI_EXPORT QgsHtmlWidgetWrapper : public QgsWidgetWrapper
     //! Sets the HTML code to \a htmlCode
     void setHtmlCode( const QString &htmlCode );
 
+    /**
+     * Returns true if the widget needs feature geometry
+     * \since QGIS 3.16
```

Comment:
3.20?

Reply:
I intended to backport to 3.16, so not sure what the policy is here.

Reply:
After second thoughts, I think it's better to write 3.20 even if I backport. I changed it